docs/src/main/asciidoc/rest-json-guide.adoc

```diff
@@ -434,6 +434,9 @@ Now, whenever a REST method is invoked, the request will be logged into the cons
 2019-06-05 12:51:04,485 INFO  [org.acm.res.jso.LoggingFilter] (executor-thread-1) Request GET /fruits from IP 127.0.0.1
 ----
 
+== CORS filter
+
+Quarkus comes with a CORS filter. Read the https://quarkus.io/guides/undertow-reference#cors-filter[Undertow Reference Documentation] to learn how to use it.
```

Comment:
Is this the right way to create a link between two Quarkus guides?

Reply:
I usually do `link:/guides/undertow-reference.html#cors-filter` to have it working when generating the Asciidoc in the main repo. `link:` is important, otherwise Asciidoc doesn't detect it properly.

But using `.html` is not a very good idea for SEO so I will leave it to you.

The `https://quarkus.io` part should be removed.